standards/certification/pipeline.md

```diff
@@ -0,0 +1,218 @@
+         application_credential_secret: "{{ clouds_conf.SUBJECT_NAME_a_ac_secret }}"
+    ```
+
+   Note that you need to replace dashes (and other special characters) by underscores in the last two lines.
+
+2. Add your subject to [Tests/config.toml](https://github.com/SovereignCloudStack/standards/blob/main/Tests/config.toml).
+   This is necessary so that your cloud will be included in the nightly tests. Add a line like so:
+
+   ```diff
+    [presets.all]
+    scopes = [
+        "scs-compatible-iaas",
+    ]
+    subjects = [
+        "gx-scs",
+   +    "SUBJECT_NAME",
+   ```
+
+   Ideally, insert your subject so that the list (after `gx-scs`) remains sorted.
+
+3. Add your subject to [compliance-monitor/bootstrap.yaml](https://github.com/SovereignCloudStack/standards/blob/main/compliance-monitor/bootstrap.yaml).
+   This is necessary to that the reports will be accepted as genuine. Add a section like so:
+
+   ```diff
+      - subject: artcodix
+        delegates:
+          - zuul_ci
+   +  - subject: SUBJECT_NAME
+   +    delegates:
+   +      - zuul_ci
+   ```
+
+   Again, insert your subject so that the list (after `gx-scs`) remains sorted.
+
+4. Finally, add secrets to [.zuul.d/secure.yaml](https://github.com/SovereignCloudStack/standards/blob/main/.zuul.d/secure.yaml).
+   This is necessary so the tests can access your cloud.
+
+   This step is the most involved, and you can always have us do it for you; in that case, please send us
+   the application credential id and secret via an encrypted channel, e.g. Matrix.
+
+   To proceed, you need `zuul-client` installed:
+
+   ```shell
+   pipx install zuul-client
+   ```
+
+   Then you can execute:
+
+   ```shell
+   $ zuul-client --zuul-url https://zuul.sovereignit.cloud/ encrypt --tenant scs --project SovereignCloudStack/standards
+   <PASTE application credential id HERE>
+   <HIT ctrl-d>
+
+   ...
+    - secret:
+        name: <name>
+        data:
+          <fieldname>: !encrypted/pkcs1-oaep
+            - ...
+
+   $ zuul-client --zuul-url https://zuul.sovereignit.cloud/ encrypt --tenant scs --project SovereignCloudStack/standards
+   <PASTE application credential secret HERE>
+   <HIT ctrl-d>
+
+   ...
+    - secret:
+        name: <name>
+        data:
+          <fieldname>: !encrypted/pkcs1-oaep
+            - ...
+   ```
+
+   Copy the parts of the respective outputs starting in the final line shown here (the one starting `-`).
+   Insert them like so:
+
+   ```diff
+   +   SUBJECT_NAME_ac_id: !encrypted/pkcs1-oaep
+   +     - ENCRYPTED_ID
+   +   SUBJECT_NAME_ac_secret: !encrypted/pkcs1-oaep
+   +     - ENCRYPTED_SECRET
+   ```
+
+   Note that you have to use the same keys as in Step 3 (that is, with special characters replaced).
+
+5. Commit your changes and open a pull request:
+
+   ```shell
+   git commit -asm "Add SUBJECT_NAME"
+   git push  # the output of this command will show you the URL for creating the pull request
```

Comment:
This will only work if you are part of the github SCS Organization. Ask the SCS team (or other members) to add you.
Otherwise you will need to create a fork in your own github project and create a PR for the SCS team to merge from there.

Reply:
add language for that

Reply:
Thx.